ddtrace/ext/git.py

```diff
@@ -76,19 +77,46 @@ def is_ref_a_tag(ref):
     return "tags/" in ref if ref else False
 
 
-def _git_subprocess_cmd(cmd, cwd=None, std_in=None):
-    # type: (Union[str, list[str]], Optional[str], Optional[bytes]) -> str
-    """Helper for invoking the git CLI binary."""
+def _git_subprocess_cmd_with_details(cmd, cwd=None, std_in=None):
+    # type: (Union[str, list[str]], Optional[str], Optional[bytes]) -> Tuple[str, str, float, int]
```

Comment:
I think I'm going to create a type for this, so that I don't have so many functions returning `Tuple[str, str, float, int]`, I just need to sort out what impact it may have if/when we backport other changes in this module to `1.x` releases.

Reply:
Make the return value a `namedtuple`? It would also be nice if we could sort the type of `cmd` to be something more homogeneous, e.g.
```suggestion
def _git_subprocess_cmd_with_details(*cmd, cwd=None, std_in=None):
    # type: (str, Optional[str], Optional[bytes]) -> Tuple[str, str, float, int]
```
but this is probably unrelated to the change and can be done in a follow-up.